Utilities/ci.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -60,7 +60,7 @@
         "repositoryURL": "https://github.com/apple/swift-syntax.git",
         "state": {
           "branch": "main",
-          "revision": "2d5575b8771c5ccd5673428d959085ddb74dbbf7",
+          "revision": "a6486790b46a4469bfb2b09d7ff6463aaecc40e5",
```

Comment:
@neonichu this is what made the macOS build fail, maybe we should try to remove this `Package.resolved` to avoid similar issues in the future?

Reply:
IIRC, without it we can't build at all, right? We can try to use an empty one.

Reply:
Trying over here https://github.com/apple/swift-tools-support-core/pull/417